pkg/shell/shell.html

```diff
@@ -2104,7 +2148,7 @@ <h4 class="modal-title">Run Image</h4>
                 </td>
               </tr>
               <tr id="containers-run-image-memory">
-                <td translatable="yes">Memory limit</td>
+                <td><label class="control-label" translatable="yes">Memory limit</label></td>
```

Comment:
Most other labels are now linked to checkboxes - for consistency, so should this one.